storage/innobase/buf/buf0dump.cc

```diff
@@ -720,6 +720,12 @@ buf_load()
 
 		buf_load_throttle_if_needed(
 			&last_check_time, &last_activity_cnt, i);
+
+#ifndef DBUG_OFF
+		if ((i+1) >= srv_buf_pool_load_pages_abort) {
+			buf_load_abort_flag = 1;
+		}
+#endif
```

Comment:
The declaration of the parameter uses `#ifdef UNIV_DEBUG`. I would use the same predicate here and in srv0srv.h.